devices/IAnalogSensorToIWear/src/IAnalogSensorToIWear.cpp

```diff
@@ -475,7 +490,7 @@ bool IAnalogSensorToIWear::attach(yarp::dev::PolyDriver* poly)
         return false;
     }
 
-    if (handler.interface || !poly->view(handler.interface) || !handler.interface) {
+    if (pImpl->handler->interface || !poly->view(pImpl->handler->interface) || !pImpl->handler->interface) {
```

Comment:
I would convert this to a negated list of `&&` so that for example the view is not executed when the pointer as already set and fails right away.